CONTRIBUTION.md

```diff
@@ -118,19 +118,20 @@ storiesOf('Molecules/Top content bar', module)
 
 ## How-to release a component
 
+**Do not use the release scripts: ~~`npm run release`~~ or ~~`npm run prerelease`~~**
+
 - Ensure you have an npm account under the `@axa-ch` organisation
 - Create a new _release branch_ that follows this pattern: `release/<component-name>`
 - Clean the main and components dependencies:
   - `npx lerna clean`
   - `rm -rf node_modules`
 - Run `npm install`
```

Comment:
I have package-lock changes at this point

Reply:
Then this is because of someone else that didn't do it correctly.